target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8068-ap3935.dts

```diff
@@ -295,55 +310,51 @@
 	};
 };
 
-&soc {
-	mdio1: mdio {
-		compatible = "virtual,mdio-gpio";
-		#address-cells = <1>;
-		#size-cells = <0>;
-
-		status = "okay";
+&mdio0 {
```

Comment:
Are you sure it's ok? mdio0 is using the gmac0 mdio instead of the gpio bitbang way... so the defined gpio won't be used... Was this tested?

Reply:
Yes, it was tested, and it does work. I realize (now) that we shouldn't need to configure the pins/gpios here. If there's any testing you'd like me to do, I have a device that runs the a current snapshot build and a build based off this branch, so happy to do so. I can update the patch to remove the extra config.

I've dumped dmesg, dtc -I dtb -O dts /sys/firmware/fdt, and some mdio-tools output from each device here: https://gist.github.com/berg/350c0d392f01636e1802dd30b03ba46c

The `.pr` files are for the device running this PR; the `.snapshot` files are for the device which netbooted the most recent snapshot as of an hour ago (r24112-67e465b32e).

Reply:
BTW, I built a new version with the extraneous gpio bits removed from the gmac mdio config and the PHYs still work fine. I can push that but it'll probably wipe out this comment thread, so I'll wait to hear from you.

Reply:
sorry for the delay. Yes push the new version. Comments are not lost so no worries!